src/redux/books/booksSlice.js

```diff
@@ -2,7 +2,50 @@ import { createSlice } from "@reduxjs/toolkit";
 
 export const booksSlice = createSlice({
   name: "book",
-  initialState: [],
+  initialState: [
+    {
+      item_id: "item1",
```

Comment:
[OPTIONAL] I think it woud be great idea to use `id: 1`, number insted of `item_id: "item1"`